authority/options.go

```diff
@@ -205,6 +206,23 @@ func WithX509SignerFunc(fn func() ([]*x509.Certificate, crypto.Signer, error)) O
 	}
 }
 
+// func WithSCEPOptions(crt *x509.Certificate, s crypto.Signer, d crypto.Decrypter) Option {
+// 	return func(a *Authority) error {
+// 		a.scepCertificate = crt
+// 		a.scepSigner = s
+// 		a.scepDecrypter = d
+// 		return nil
+// 	}
+// }
+
+func WithFullSCEPOptions(options *scep.Options) Option {
```

Comment:
Add comment and add something like:
```go
// This feature is EXPERIMENTAL and might change at any time.
```

Reply:
Fixed in c0fbace88227fcfeb5f92415fa6158e4859638b3.